### ISSUE TITLE
How many camera images in total in the training set?

### ISSUE THREAD

**Issue by wdf19961118** [waymo-research/waymo-open-dataset#553]
After parsing the tfrecord files in training set, I found that there are 790405(158081*5) images in the training set, and only 61475(12295*5) images have segmentation label. Besides, validation set has 199935(39987*5) images in which 9405(1881*5) images have segmentation label. However, it doesn't match the number in the paper(Waymo Open Dataset: Panoramic Video Panoptic Segmentation).
![image](https://user-images.githubusercontent.com/30190270/186605954-28f9c045-e0da-4ed1-b10e-3df3c8e5d1fc.png)

**Comment by alexzzhu** [comment]
Hi there, sorry for the late response. We started out with 100k images to be labeled, but it seems that some have been lost due to data corruption on the way. We are planning on updating the dataset later this month / early next month to recover more of the labels. In the end, there will be a few labels lost, but we expect this to be <2k for the training set and < 100 for the validation set.
